### PR TITLE
Update Override.txt

### PR DESCRIPTION
**Pull request by Onewhoknowsnocode**
# Update Override.txt - Onewhoknowsnocode(Zosrin)
-Basis for items, techniques, monster info  & Random Resizing 
-Check Lines  775-782 for some Regular Expression Suggestions

### CODE REVIEW

File: Translation/en/Text/Override.txt [hunk 1]
```diff
@@ -703,3 +712,168 @@ EA开放=EA Open
...
+##Drops Name
+##Drops Descriptions
+##Ignore Repeats
+
+# Fierce Tiger
+猛虎=<size\=16>Fierce Tiger</size>
+撕咬=<size\=14>Bite</size>
+百兽之王=<size\=14>King of All Beasts</size>
+虎扑=<size\=14>Tiger's Mark</size>
+虎须=<size\=14>Tiger Wiskers</size>
```
Comment (pipja) [inline Translation/en/Text/Override.txt]: Whiskers

File: Translation/en/Text/Override.txt [hunk 2]
```diff
@@ -703,3 +712,168 @@ EA开放=EA Open
...
+虎的毛须，长而韧。=Wiskers of a tiger are long and tough.
```
Comment (pipja) [inline Translation/en/Text/Override.txt]: Same as above

File: Translation/en/Text/Override.txt [hunk 3]
```diff
@@ -703,3 +712,168 @@ EA开放=EA Open
...
+灵根：=<size\=13>Roots:</size>
+强度：=<size\=13>Strgth:</size>
+
+掉落=<size\=16>Drops</size>
+设为灵宠=<size\=16>Set as\nSpirit Pet</size>
+
+
+### Monster Drop Classification
+
+皮毛-怪物材料=<size\=12>Skin & Hair \n -Monster Material</size>
```
Comment (pipja) [inline Translation/en/Text/Override.txt]: Pelt & Feather(?)?

File: Translation/en/Text/Override.txt [hunk 4]
```diff
@@ -703,3 +712,168 @@ EA开放=EA Open
...
+拥有奇妙作用的虎骨，受到很多男性修士青睐。=A Tiger bone, which has wonderful effects, is favored by many male cultivators.
+
+# F-Tiger Cub
+幼虎=<size\=16>F-Tiger Cub</size>
+
+# White Tiger
+白虎=<size\=16>White Tiger</size>
+
+# W-Tiger Cub
+幼白虎=<size\=16>W-Tiger Cub</size>
```
Comment (pipja) [inline Translation/en/Text/Override.txt:949]: What's W? White?

**Review by pipja — APPROVED** [review]

File: Translation/en/Text/Override.txt (hunk 4 above)
Comment (Onewhoknowsnocode) [reply Translation/en/Text/Override.txt:949]: Yeah, Also Dev oversite. 
幼白虎 is also used to refer to the Falling source for the tigers aswell

**Pull request merged**